show/main.py

```diff
@@ -1542,13 +1503,22 @@ def tacacs():
                 output += ('               %s %s\n' % (key, str(entry[key])))
     click.echo(output)
 
+
 #
-# 'mirror_session' command  ("show mirror_session ...")
+# 'mirror' group ###
 #
-@cli.command()
+
+@cli.group(cls=AliasedGroup, default_if_no_args=False)
+def mirror():
+    """Show mirroring (Everflow) information"""
+    pass
+
+
+# 'session' subcommand  ("show mirror session")
+@mirror.command()
 @click.argument('session_name', required=False)
 @click.option('--verbose', is_flag=True, help="Enable verbose output")
-def mirror_session(session_name, verbose):
+def session(session_name, verbose):
```

Comment:
Again, I don't think you meant to modify this. I believe your code was out-of-date.

Reply:
My bad. I thought I had grabbed the most recent version of the file but it looks like I missed a step.   I've updated this now.